post_installation_configuration/multi-architecture-configuration.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+:context: multi-architecture-configuration
+[id="post-install-multi-architecture-configuration"]
+= Configuring a multi-architecture cluster
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+A multi-architecture cluster is a cluster that supports worker machines with different architectures. You can deploy a multi-architecture cluster by creating an Azure installer-provisioned cluster using the multi-architecture installer binary. For Azure installation, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing on Azure with customizations].
```

Comment:
Just a nit for using the name of the doc being linked to:
```suggestion
A multi-architecture cluster is a cluster that supports worker machines with different architectures. You can deploy a multi-architecture cluster by creating an Azure installer-provisioned cluster using the multi-architecture installer binary. For Azure installation, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing a cluster on Azure with customizations].
```

Reply:
Ooo, nice catch! Thank you for the review!